src/Main.java

```diff
@@ -1,5 +1,98 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Привет, мир!");
+        // задача №1
+        int a = 1000000;
+        System.out.println("Значение пременной a " + " с типом int " + " равно " + a);
+        byte b = 125;
+        System.out.println("Значение пременной b " + " с типом byte " + " равно " + b);
+        short c = 30000;
+        System.out.println("Значение пременной c " + " с типом short " + " равно " + c);
+        long d = 250000000;
+        System.out.println("Значение пременной d " + " с типом long " + " равно " + d);
+        float e = 3.25F;
+        System.out.println("Значение пременной e " + " с типом float " + " равно " + e);
+        double i = 2.67;
+        System.out.println("Значение пременной i " + " с типом double " + " равно " + i);
+        // задача № 2
+        double w = 27.12;
+        System.out.println(w);
+        long q = 987678965549L;
```

Comment:
```suggestion
        long q = 987_678_965_549L;
```
для читаемости лучше разделять большие числа